src/Queue.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Equip\Queue;
+
+use Equip\Queue\Driver\DriverInterface;
+
+class Queue
+{
+    /**
+     * @var DriverInterface
+     */
+    private $driver;
+
+    /**
+     * @param DriverInterface $driver
+     */
+    public function __construct(DriverInterface $driver)
+    {
+        $this->driver = $driver;
+    }
+
+    /**
+     * Add message to queue
+     *
+     * @param string $queue
+     * @param string $name
+     * @param array $data
+     * @param array $meta
```

Comment:
`$data` is application specific, such as `user_id`.

`$meta` is queue specific, such as the ability to run a job a specific amount of time if it fails.